use-fish

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env fish
+# Usege:
+#
+#   source <(curl https://raw.githubusercontent.com/k1LoW/sheer-heart-attack/master/use)
+#
+# Reference:
+#   https://github.com/goreleaser/get
+set SHA_GOOS linux
+set SHA_EXT tar.gz
+set SHA_ARCH amd64
+if test (uname -s) = "Darwin"
+  set SHA_GOOS darwin
+  set SHA_EXT zip
+end
+
+set SHA_RELEASES_URL "https://github.com/k1LoW/sheer-heart-attack/releases"
+test -z "$SHA_TMPDIR" && set SHA_TMPDIR (mktemp -d)
+set SHA_ARCHIVE ""$SHA_TMPDIR"sheer-heart-attack."$SHA_EXT""
+
+function last_version
+  curl -sL -o /dev/null -w '%{url_effective}' ""$SHA_RELEASES_URL"/latest" |
+    rev |
+    cut -f1 -d'/'|
+    rev
+end
+
+function download
+  test -z "$SHA_VERSION" && set SHA_VERSION (last_version)
```

Comment:
`&&` and `||` are available on fish 3.0+.
it is better to use `;and` and `;or` when considering compatibility.